### PR TITLE
Publish to NPM on new releases

### PR DESCRIPTION
**Pull request by qwerty287**
Add an action to publish new releases to NPM. I generated and added an access token to the secrets.

### CODE REVIEW

File: .github/workflows/Publish.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 18.x
+          registry-url: https://registry.npmjs.org/
+      - run: npm install
+      - run: npm publish --access public
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
```
Comment (nagmat84) [inline .github/workflows/Publish.yml:19]: Where and who stores this secret such that Github can access NPM? And whose account is used for that?

Comment (qwerty287) [reply .github/workflows/Publish.yml:19]: It's for my account. Secrets are under https://github.com/LycheeOrg/basicContext/settings/secrets/actions.

**Review by nagmat84 — APPROVED** [review]
I never was patient enough to figure out how to do it. But it looks reasonable. Probably we should do something similar for Composer and Packagist for our PHP packages php-exif and Co.

**Comment by qwerty287** [comment]
Composer updates itself I think. But I'll do the same for Leaflet.Photo and basicModal.

File: .github/workflows/Publish.yml (hunk 1 above)
Comment (nagmat84) [reply .github/workflows/Publish.yml:19]: Can we somewhere document that? Probably the best option is as a comment in the script above the line.

Comment (nagmat84) [reply .github/workflows/Publish.yml:19]: Too late it is already merged. :grinning:

Comment (qwerty287) [reply .github/workflows/Publish.yml:19]: Yes, I'll add a new PR.

**Pull request merged**